.circleci/config.yml

```diff
@@ -153,7 +154,7 @@ jobs:
             export SCIPY_AVAILABLE_MEM=1G
             # Try to limit per-process GC memory usage
             export PYPY_GC_MAX=900MB
-            pypy3 runtests.py -- -rfEX -n 2 --durations=30 --timeout=45 --timeout-method=thread -v
+            pypy3 runtests.py -- -rfEX -n 2 --durations=30 --timeout=45 --timeout-method=thread
```

Comment:
The `-v` is there to help triage crashes, but we can remove it for now. #9980 should also help make things more readable.